### ISSUE TITLE
Implementation of Suervised Baseline

### ISSUE THREAD

**Issue by omegaiota** [isantesteban/snug#18]
Hi, 
Thank you for the very interesting work. I have a  question regarding the baseline implementation shown in this  [comparison video](http://mslab.es/projects/SNUG/contents/snug_comparison_v4.mp4):
I wonder if the supervised baseline model also outputs a regressor for the garment mesh, then uses skinning to compute the final computation? Is it correct that the only difference between SNUG and this suerpvised baseline is the loss function that is used to train the model?

Thank you!

**Comment by dancasas** [comment]
@omegaiota yes, you are right. The supervised baseline uses a loss based on L2 distance to the ground truth positions of the garment vertices that were simulated using state-of-the-art cloth simulation (i.e., it requires supervision at the vertex level). SNUG does not require such supervision, since its loss is based on general physics properties that do not need to be precomputed.
